src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.component.spec.ts

```diff
@@ -397,7 +398,7 @@ describe('Component: GeneComparisonToolComponent', () => {
       expect(
         element.querySelector(TOGGLE_CLASS)?.querySelector('input')?.checked
       ).toBeTrue();
-      expect(component.getUrlParam('significance')[0]).toEqual(
```

Comment:
this used to [0] since significance used to be an array.  This has been changed because it does not make sense for significance to be an array.